### ISSUE TITLE
微调脚本报错

### ISSUE THREAD

**Issue by lime-s** [IDEA-Research/Rex-Omni#29]
我使用4090显卡(24GB)，由于显存限制我将--per_device_train_batch_size 设置为 2，将--gradient_accumulation_steps设置为32，以保持batch_size相同，在加载过程中：
>>>>>>> Step4: Start Training <<<<<<<
Gradient accumulation steps mismatch: GradientAccumulationPlugin has 1, DeepSpeed config has 32. Using DeepSpeed's value.
最终还是爆显存，请问我该怎么解决，是禁用deepseed，还是只能更换40GB显存的显卡

**Comment by Mountchicken** [comment]
batch_size 不一定要保持一致。试一下 --per_device_train_batch_size==1， --gradient_accumulation_steps==4然后用 zero3.json，zero3 还不行就试试 zero3_offload.json

**Comment by xlg-go** [comment]
> 我还没跑通，估计全参数调的话24G不够，可以试试LORA

您好，请问lora了吗？
